Bothelper/Program.cs

```diff
@@ -92,8 +390,18 @@ static void Main(string[] args)
                     isContinueWork = false;
                     Console.Clear();
                 }
+                
+
+
             }
+            
+            
 
         }
+        
+
```

Comment:
pls remove empty spaces (what are they leaving for?)
<img width="1024" height="1480" alt="image" src="https://github.com/user-attachments/assets/8ce37cf9-c9e0-477c-80b6-bba0fffe795a" />